### DISCUSSION TITLE
How integrated Material UI Select component with RHF?

### DISCUSSION THREAD

**Discussion by vicasas** [react-hook-form/react-hook-form#3882] [Q&A] [answered]
I am building a form that has a selectable field of type `Select` to which I send its inputRef the same as the others but it does not work.

I have reviewed quite a bit on the subject, and it is a known problem. But I don't know if today there is a solution for this error, the only thing I see is the use of the RHF Controller component.

**Comment by bluebill1049** [comment, marked as answer]
have you seen this section: https://react-hook-form.com/api#Controller there is a csb which may be helpful: https://codesandbox.io/s/react-hook-form-v6-controller-qsd8r

**Reply by vicasas** [reply]
I understand, thank you!
